annotator-core/src/main/java/edu/ucr/cs/riple/core/util/Utility.java

```diff
@@ -89,14 +88,12 @@ public static void executeCommand(Config config, String command) {
    * @param path Path to the file.
    */
   public static void clearFileContentsAtPath(Path path) {
-    if (Files.exists(path)) {
-      try (BufferedWriter writer =
-          Files.newBufferedWriter(
-              path, Charset.defaultCharset(), StandardOpenOption.TRUNCATE_EXISTING)) {
-        writer.flush();
-      } catch (IOException e) {
-        throw new RuntimeException("Could not clear content of file: " + path, e);
-      }
+    try {
+      Files.deleteIfExists(path);
+      Files.createDirectories(path.getParent());
+      Files.createFile(path);
```

Comment:
I think this is modifying logic introduced in #220.  Still not sure why we are creating an empty file here.

Reply:
Yes, this is fixed in #220 now.